websites/E/Emby/presence.ts

```diff
@@ -408,59 +433,52 @@ function getUserId(): string {
 			localStorage.getItem("servercredentials3")
 		).Servers;
 
-		// server id available on browser location
-		if (location.hash.indexOf("?") > 0) {
-			for (const param of location.hash.split("?")[1].split("&")) {
-				if (param.startsWith("serverId")) {
-					for (const server of servers)
-						if (server.Id === param.split("=")[0]) return server.UserId;
-				}
-			}
-		} else return servers[0].UserId;
+		return (
+			servers.length === 1
+				? servers[0]
+				: servers.find(
+						(s: { Id: string }) =>
+							s.Id ===
+							new URLSearchParams(location.hash.split("?")[1]).get("serverId")
```

Comment:
```suggestion
							new URLSearchParams(location.search).get("serverId")
```

Reply:
The search (GET) parameters of the URL are in part of the hash, see image below.

![image](https://user-images.githubusercontent.com/22194879/171866771-8728adcd-b7f2-4fc1-8b47-780be1a3b6d9.png)